packages/generator-volto/yarn.lock

```diff
@@ -8460,12 +8460,12 @@ __metadata:
   languageName: node
   linkType: hard
 
-"prettier@npm:2.0.5":
-  version: 2.0.5
-  resolution: "prettier@npm:2.0.5"
+"prettier@npm:3.0.2":
+  version: 3.0.2
+  resolution: "prettier@npm:3.0.2"
   bin:
-    prettier: bin-prettier.js
-  checksum: 7f89d2f5d1a1a15a7bf200556b8c5395913d0119fe52a3fba51ab116695d584652d37fdb7d86e0919bfe36b22649afcf222eaca00a1065eb486b7b4cf3062eff
+    prettier: bin/prettier.cjs
+  checksum: 118b59ddb6c80abe2315ab6d0f4dd1b253be5cfdb20622fa5b65bb1573dcd362e6dd3dcf2711dd3ebfe64aecf7bdc75de8a69dc2422dcd35bdde7610586b677a
```

Comment:
Again, I don't know why you changed this verion.

Reply:
Sorry to say @stevepiercy, I have not changed anything showing above. I just simply added docs and did nothing. I don't know how this has happened.

Reply:
hmm, so this was not you, but someone masquerading as you?

https://github.com/plone/volto/blame/17e96908d99696b3e631184c733c0fa4edc8dac1/packages/generator-volto/yarn.lock#L8467-L8468

It's just a mistake, no big deal. If you don't know why something changes, then do not `git add` the changed files and push them. Also pay careful attention to your editor if it suggests that you add files to a commit that you did not intentionally change. Finally, do not just `git add` from the command line because that adds every file that you changed, added, and deleted, whether you realize it or not.